recommended-setup.js

```diff
@@ -0,0 +1,121 @@
+const gulp = require('gulp');
+const watch = require('gulp-watch');
+const babel = require('gulp-babel');
+const scssTargets = require('./Tasks/scss-targets');
+const jsTargets = require('./Tasks/js-targets');
+const copyAssets = require('./Tasks/copy-assets');
+
+const defaultAssetsBasePath = './Assets/';
+const defaultDistBasePath = './wwwroot/';
+const defaultStylesAssetsBasePath = defaultAssetsBasePath + 'Styles/';
+const defaultScriptsAssetsBasePath = defaultAssetsBasePath + 'Scripts/';
+
+function setupRecommendedScssTasks(
+    stylesDistBasePath = defaultDistBasePath + 'css/',
+    stylesAssetsBasePath = defaultStylesAssetsBasePath) {
+    gulp.task('build:styles', scssTargets.build(stylesAssetsBasePath, stylesDistBasePath));
+    gulp.task('clean:styles', scssTargets.clean(stylesDistBasePath));
+    gulp.task('watch:styles', () => watch(stylesAssetsBasePath + '**/*.scss', { verbose: true }, gulp.series('build:styles')));
+    gulp.task('default', gulp.series('build:styles'));
+    gulp.task('clean', gulp.series('clean:styles'));
+
+    return this;
```

Comment:
Ez a `return this` a chainghez kell, ugye? Viszont a `setupRecommendedScssAndJsTasksAndVendorsCopyAssets`-ban és a `setupRecommendedScssAndJsTasksAndCopyAssets`-ban nincs ott. Szándékos?

Reply:
Igen, és nem :|, de már igen :).